src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/config/HmppsAssessForEarlyReleaseApiExceptionHandler.kt

```diff
@@ -52,6 +53,17 @@ class HmppsAssessForEarlyReleaseApiExceptionHandler {
       ),
     ).also { log.info("No resource found exception: {}", e.message) }
 
+  @ExceptionHandler(PrisonerNotFoundException::class)
+  fun handlePrisonerNotFoundException(e: PrisonerNotFoundException): ResponseEntity<ErrorResponse> = ResponseEntity
```

Comment:
we use `EntityNotFoundException` in a few places, would probably be better to not have to handle different types of missing resources -so could have a new generic exception or use EntityNotFoundException

Reply:
Did wonder, I just struggled over weather it was an Entity, So how about ItemNotFoundException?